db/db.js

```diff
@@ -4,10 +4,13 @@
 
 module.exports = function () {
     const mongoose = require('mongoose'),Admin=mongoose.mongo.Admin;
-    let mongoURL =process.env.MONGO_URL || 'localhost';
-    let mongoPORT = process.env.MONGO_PORT|| '27017';
-    let dev_db_url = `mongodb://${mongoURL}:${mongoPORT}/climateTree`;
-    let mongoDB = process.env.MONGODB_URI || dev_db_url;
-    mongoose.connect(mongoDB);
+    let mongoPORT = process.env.MONGO_PORT|| '10255';
```

Comment:
We do not want everyone to use deployed instance locally. Can you use environment variables.